### PR TITLE
#143 add loadFixtureFiles with Nelmio alice

### PR DESCRIPTION
**Pull request by nicolas-bastien**
Hi @lsmith77 ,

So here is my code. I added the foreign keys check as we need it on our project with the platform test as you suggest : https://github.com/liip/LiipFunctionalTestBundle/issues/145

### CODE REVIEW

**Comment by lsmith77** [comment]
One thing that I am wondering is in light of this https://github.com/liip/LiipFunctionalTestBundle/pull/136 quite sensible refactoring to split out the fixture loading into a separate class, maybe we should do this with this feature already?

Or better yet, are you motivated to also wrap up the above noted PR?

**Comment by nicolas-bastien** [comment]
When doing this PR I had the same idea of refactoring.

Maybe we can merge this first cause has this is not a big change.

then I'll help @thomas-koch-comsolit on is PR cause I think his refactoring needs a little more work.
Like @stof notes, documentation, services loading....

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
         "php": ">=5.3.2",
         "symfony/framework-bundle": "~2.0",
         "symfony/browser-kit": "~2.0",
-        "doctrine/common": "2.*"
+        "doctrine/common": "2.*",
+        "nelmio/alice": "~1.7"
```
Comment (lsmith77) [inline composer.json]: this should imho be an optional dependency. in this sense there should maybe then also be an explicit check in the code if the class exists before making an instance.

**Comment by lsmith77** [comment]
yeah, its ok for me to merge it in (see my one comment).

**Comment by nicolas-bastien** [comment]
@lsmith77 done

**Pull request merged**